templates/cloudflare-pages/server.ts

```diff
@@ -7,6 +7,6 @@ const handleRequest = createPagesFunctionHandler({
   getLoadContext: (context) => context.env,
 });
 
-export function onRequest(context) {
+export function onRequest(context: EventContext<any, any, any>) {
```

Comment:
Since `onRequest` just calls `handleRequest`, I'm just going to replace it with the implementation of `onRequest`, as we do that in other `server.ts` files as well